cmd/install.go

```diff
@@ -14,8 +14,8 @@ import (
 // nolint:gosec
 func init() { // nolint:gochecknoinits,gocyclo
 
-	var dbSeedCmd = &cobra.Command{
-		Use:   "db-seed  [environment]",
+	var installCmd = &cobra.Command{
+		Use:   "install [environment]",
 		Short: "fill the database with required data",
```

Comment:
We should be more explicit than "fill the database with required data". Actually, in normal cases, you create the groups then you put the IDs into the config. It is strange to have a "static" ID in the config and the let the app creating the entry (hoping there is no conflict). 
So we should be more explicit. For the moment as it is just for that, let's explain it full (you can use the "Long" key for that: 
```
Long: `In case the root group id's specified into the config file are not existing
            or have missing relations, creates them all (groups,  groups_groups, and groups_ancestors)`
```

Reply:
added the long description